Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -1686,7 +1686,7 @@
       Dead: 0
     baseDecayRate: 0.04
   - type: Hunger
-    currentHunger: 25   # spawn with Okay hunger state
```

Comment:
This name change definitely sucks, but I don't know a way around it.

It would be good for `currentHunger` or `startingHunger` to be the field in the yaml, but it would, I think, not be good to  have either of those names show up in-game in something like ViewVariables.

Reply:
> This name change definitely sucks, but I don't know a way around it.
> 
> It would be good for `currentHunger` or `startingHunger` to be the field in the yaml, but it would, I think, not be good to have either of those names show up in-game in something like ViewVariables.

`[ViewVariables(VVAccess.ReadOnly)]` it.

Also I'm looking at the code, does setting this data field from YAML even work? It looks like it would always get replaced by the randomized amount selected in `OnMapInit`. (Note that this doesn't appear to be an issue with your PR, just an observation).

Reply:
It looks like this was changed with the last rewrite of all of this:
https://github.com/space-wizards/space-station-14/pull/14939/files#diff-7ff03c169eb454aed40cb0979bd1d81f65ce0db09590d25191682c783e7ac344L131

Should I get rid of the fields in the yaml, since they don't do anything now?

Reply:
Your choice I suppose